
I'm growing this variable to match with Linux's because too many packages
assume it too big and fail to build or run.  -- rmh

--- sys.old/sys/syslimits.h	2003-09-10 21:08:16.000000000 +0200
+++ sys/sys/syslimits.h	2004-08-20 03:54:22.000000000 +0200
@@ -47,7 +47,7 @@
  * Do not add any new variables here.  (See the comment at the end of
  * the file for why.)
  */
-#define	ARG_MAX			65536	/* max bytes for an exec function */
+#define	ARG_MAX		       131072	/* max bytes for an exec function */
 #ifndef CHILD_MAX
 #define	CHILD_MAX		   40	/* max simultaneous processes */
 #endif
